### PR TITLE
check GitVersion to detect managed clusters and activate node-agent

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

**Review by amirmalka — APPROVED** [review]

**Comment by dwertent** [comment]
@matthyx What if I'm running an unmanaged cluster but I still want relevancy to run. How can I do that?

**Comment by matthyx** [comment]
> @matthyx What if I'm running an unmanaged cluster but I still want relevancy to run. How can I do that?

you can't... should I add a forceEnable to relevancy?

**Comment by dwertent** [comment]
Great!
We will merge this after we see the system tests are stable. Because we will need the system tests to run 
```
--set capabilities.relevancy=enable
```
instead of enabling the `node-agent`.

**Comment by matthyx** [comment]
> Great! We will merge this after we see the system tests are stable. Because we will need the system tests to run
> 
> ```
> --set capabilities.relevancy=enable
> ```
> 
> instead of enabling the `node-agent`.

works on my PC btw 😅

File: charts/kubescape-cloud-operator/values.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ ksNamespace: kubescape
 ksLabel: kubescape
 createKubescapeServiceAccount: true # TODO: move to kubescape
 
+capabilities:
+  relevancy: detect # enable/disable/detect
```
Comment (dwertent) [inline charts/kubescape-cloud-operator/values.yaml:10]: Default should be `disable`

**Review by dwertent — CHANGES_REQUESTED** [review]

**Pull request merged**